pkg/project/file.go

```diff
@@ -80,10 +80,18 @@ func FindFile(fp string) (string, bool, error) {
 	return "", false, nil
 }
 
-// fileExists checks if a file exist and is not a directory.
+// fileExists checks if a file or directory exist.
 func fileExists(fp string) bool {
 	_, err := os.Stat(fp)
-	return err == nil || os.IsExist(err)
+	if err == nil {
+		return true
+	}
+
+	if os.IsNotExist(err) {
+		return false
+	}
+
+	return false
```

Comment:
Out of interest, what was wrong with the original implementation here?

In the current form, the `os.IsNotExist(err)` condition, does not really make sense, as the two last returns are both `false`.

Reply:
The current implementation looks good though. Reverting it here.